app/renderer/components/navbar.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
     mounted() {
         Material.checkFirstTimeTap(".tap-target");
     },
-    methods: {
+  methods: {
```

Comment:
wrong alignment

Reply:
[js-beautify](https://www.npmjs.com/package/js-beautify) is a good package for javascript automatic formatting, try that if your editor is not formatting js properly